.gitignore

```diff
@@ -1,10 +1,23 @@
+# OS X
+.DS_Store
```

Comment:
Thx :) I can't take full credit. This just seemed to be the norm in similar projects I viewed for comparison when I did "new project", so I copy pasted. Plus I've removed more than a few `.DS_Store` files over time.